townsfolk/investigative/juror

```diff
@@ -0,0 +1,9 @@
+**Juror** | Townsfolk Group
+__Basics__
+The Jurors have a secret channel where they can communicate among themselves securely. On alternate days, the Jury votes to investigate a player.
+If the Jury successfuly investigates a non-town role, it is announced.
```

Comment:
Not a fan of just announcing a role, even if it's only a role and not a complete check. I like the idea of the Jurors but this is not really neccesary

Reply:
Yeah, actually I think I agree. I think this could still work without the announcement.

Reply:
I felt like the announcing mechanic was interesting because it lets all parties have an understanding of where the game is and shift the game as a whole.

Its an obvious advantage to town as:
1.  Townies know that said role has been confirmed and can use that info accordingly
2.  A juror/jury can effectively confirm themselves without necessarily revealing everyone in the jury
3.  Makes way for an interesting mechanic where you have discussion in town hall on who the jury would check. Obviously
there's no guarantee the Jury would've followed through with public requests.

Its advantageous to wolves since:
1.  Town can no longer hide the fact that they know a person is evil, which means they cant _pretend_ to trust someone to keep their faction under a false sense of security.
2.  Revealed roles will have to change their gameplay, becoming bait/sacrifice or becoming an ideal member to go scouting for non-pack members.

Also, the jury would have to play differently and uniquely than other investigative roles and would really be involved with public census. They have to choose carefully as to not cause panic, confusion and tip off the wolves while also trying not to out themselves. Further, in a more meta sense, town-hall activity would increase and bring people out of conspiracy channels.
Removing the announcement thing just makes them fortune teller bakers in all honesty.

Reply:
You're brought up a bunch of things I didn't think of. I'm convinced.